src/main/scala/devices/spi/TLSPI.scala

```diff
@@ -120,7 +123,13 @@ class SPITopModule(c: SPIParamsBase, outer: TLSPIBase)
       RegField.r(1, ip.txwm,
       RegFieldDesc("txwm_ip","Transmit watermark interupt pending", volatile=true)),
       RegField.r(1, ip.rxwm,
-      RegFieldDesc("rxwm_ip","Receive watermark interupt pending", volatile=true)))))
+      RegFieldDesc("rxwm_ip","Receive watermark interupt pending", volatile=true)))),
+
+    SPICRs.extradel -> RegFieldGroup("extradel",Some("delay from the sck edge"),Seq(
+      RegField(c.divisorBits, ctrl.extradel.coarse,
+      RegFieldDesc("extradel_coarse","Coarse grain sample delay", reset=Some(0))),
+      RegField(5, ctrl.extradel.fine,
```

Comment:
shouldn't this 5 be `c.fineDelaySelectWidth`

Reply:
Ah, I think I made this change in another repo and forgot to merge before pushing this - will update.